polyfills/Array/prototype/includes/tests.js

```diff
@@ -16,6 +29,10 @@ it('has correct argument length', function () {
 	proclaim.equal(Array.prototype.includes.length, 1);
 });
 
+ifSupportsDescriptorsIt('is not enumerable', function () {
```

Comment:
We have to use this because in environments where descriptors are not supported, you can not create a non-enumerable property.